__tests__/server.test.js

```diff
@@ -105,6 +105,34 @@ describe("GET /api/articles", () => {
         expect(body.articles).toBeSortedBy("created_at", { descending: true });
       });
   });
+  test("should return only the articles that have the correct topics according to the query", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toBeGreaterThan(0);
+        body.articles.forEach((article) => {
+          expect(article.topic).toBe("mitch");
+        });
+      });
+  });
+  test("should return a 404 error if we're being passed a valid topic with no articles", () => {
+    return request(app)
+      .get("/api/articles?topic=paper")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.message).toBe("No articles available with that topic");
```

Comment:
If the topic _exists_ in the database then this isn't a 404, it's a `200`. The response should just be an empty array, similar to if there is an article with no comments from ticket 6